src/Taps.h

```diff
@@ -32,6 +34,8 @@ class Tap : public AbstractModule
 
     std::unique_ptr<InputStream> instantiate(const Configurable *config, const Configurable *filter, std::string input_name);
 
+    bool tags_validate_yaml(const YAML::Node &tag_yaml, bool all);
```

Comment:
i think we are not validating here, but matching, right? so `tags_match_selector_yaml` maybe.

but i actually wonder if we should instead make this a method of Configurable. then we could use config_set_yaml, and we'd be able to reuse this matching functionality